githubapp/ldap.py

```diff
@@ -14,7 +14,10 @@ def __init__(self):
         self.LDAP_GROUP_BASE_DN = os.environ['LDAP_GROUP_BASE_DN']
         self.LDAP_GROUP_FILTER = os.environ['LDAP_GROUP_FILTER']
         self.LDAP_GROUP_MEMBER_ATTRIBUTE = os.environ['LDAP_GROUP_MEMBER_ATTRIBUTE']
-        self.LDAP_BIND_USER = os.environ['LDAP_BIND_USER']
+        if 'bind_user' in os.environ:
```

Comment:
```suggestion
        if 'LDAP_BIND_USER' in os.environ:
```

that works... do we want to standardize on one or the other? I chose `USER` arbitrarily

Reply:
actually, with `user@domain.com` it wouldn't make sense to call it a `DN`... maybe we just leave it with both

Reply:
Yeah I don't mind either way as long as add a comment about it